Source/Core/VideoBackends/Software/Tev.cpp

```diff
@@ -812,28 +812,28 @@ void Tev::SetRegColor(int reg, int comp, bool konst, s16 color)
 
 void Tev::DoState(PointerWrap &p)
 {
-	p.DoArray(Reg, sizeof(Reg));
+	p.DoArray(Reg);
 
-	p.DoArray(KonstantColors, sizeof(KonstantColors));
-	p.DoArray(TexColor,4);
-	p.DoArray(RasColor,4);
-	p.DoArray(StageKonst,4);
-	p.DoArray(Zero16,4);
+	p.DoArray(KonstantColors);
```

Comment:
`KonstantColors` consists of `s16`s, so doesn't changing from `sizeof` mean that savestates now have a different layout, requiring an update of the savestate version number?